source/support.tex

```diff
@@ -2127,7 +2127,7 @@
 terminates by throwing an exception~(\ref{expr.new}).
 \end{itemdescr}
 
-\indexlibrary{\idxcode{new}!\idxcode{operator}}%
+\indexlibrary{\idxcode{delete}!\idxcode{operator}}%
```

Comment:
Actually, don't worry. This is definitely an improvement, and it's consistent. Thanks.